src/lib/types.ts

```diff
@@ -372,6 +376,10 @@ export type PlotOptions = {
      */
     overlay: Snippet;
     facetAxes: Snippet;
+    /**
+     * A background color
+     */
+    background: string;
```

Comment:
Setting a background color via prop is not supported. We should remove this type and adjust the related test.

```suggestion
```

Reply:
But I can also remove it after merging, actually!